precedence.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+    "use strict";
+
+    function Precedence() {
+        this.numbers = [];
+        this.operations = [];
+        this.turnOn = false;
+        this.finalResult = 0;
```

Comment:
You do not need this field either, just use local variable in *calculateResult()* method.